lib/utils/createModifier.js

```diff
@@ -17,9 +17,11 @@ function createModifier({ modelClass, modifier, modifiers, args }) {
     let modify = null;
 
     if (isString(modifier)) {
-      modifier = modifiers[modifier] || modelModifiers[modifier];
+      modify = modifiers[modifier] || modelModifiers[modifier];
```

Comment:
@koskimas this was replacing `modifier` here with `undefined`, leading to wrong exception also that mentioned `undefined` instead of the actually missing modifier name.